docs/docs/events/custom/overview.md

```diff
@@ -0,0 +1,8 @@
+# Custom Events
+
+Tracee comes with lots of events, but you can extend it with events specific to your use case. There are two ways to extend Tracee with your own events:
+
+1. [Go](./golang.md)
+2. [Rego](./rego.md)
+
```

Comment:
We should add something about `--signatures-dir` that can be used to point to a user defined directory containing signatures

Reply:
But this is done in `rego`already, why do it again? https://aquasecurity.github.io/tracee/v0.14/docs/events/custom/rego/